src/main/kotlin/org/jetbrains/bio/io/Bed.kt

```diff
@@ -306,6 +306,8 @@ data class BedFormat(
  * Reads [BedEntry]-s from the supplied [reader].
  *
  * Set [stringency] to your preferred mode before starting to parse. See [Stringency] for mode explanation.
```

Comment:
Actually not very clear from the doc that it is ok to change property value while parsing (after constructor was called).  Could we add some small code example how to parse in strict mode? Normally we don't call BedParser constructor manually and use `parse()` static method.

Also why `stringency` is a constructor argument? Seems we never customize it from the constructor call. Is it better to fix the only one way of dealing with this property, e.g. just change it via getter/setter, not in the constructor?

Reply:
Added code example, removed stringency from the constructor.